src/hotspot/share/opto/type.hpp

```diff
@@ -216,15 +216,15 @@ class Type {
   // Create a new hash-consd type
   static const Type *make(enum TYPES);
   // Test for equivalence of types
-  static int cmp( const Type *const t1, const Type *const t2 );
```

Comment:
Was it required to remove the `consts` here?

Reply:
Since the second `const` is referring to the `t1/t2` it means that the parameters can't be modified, which IIRC in the header would be functionally the same as not having the additional const. I changed it in type.cpp as well because looking at other parts of the code `const Type* val` is used a lot more often than `const Type* const val` for arguments.